Application/EdFi.Common/EdFi.Common.csproj

```diff
@@ -13,7 +13,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Autofac.Extras.DynamicProxy" Version="6.0.1" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.1.1" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.1.4" />
```

Comment:
This version was bumped because of a Moderate security vulnerability in 5.1.1. Automated detection of the vulnerable version caused all builds that depend on this project to fail, including the projects updated in this PR.